GDJS/Runtime/Electron/main.js

```diff
@@ -46,6 +43,10 @@ function createWindow() {
   });
 }
 
+// Explanation taken from: newIDE/electron-app/app/main.js
+// Disable this as we rely heavily on `fs` in the renderer process.
```

Comment:
I would enter a complete description here too, because these part of the codebase are relatively independent :)

```
// Should be set to true, which will be the default value in future Electron
// versions, but then causes an issue on Windows where the `fs` module stops	
// working in the renderer process after a reload.
// See https://github.com/electron/electron/issues/22119
// For now, out of caution, disable this as we rely heavily on `fs` in the 
// renderer process.
```

I added `after a reload` because it's less likely to happen in a game, but still.